### ISSUE TITLE
Bug: Redirect imports not working

### ISSUE THREAD

**Issue by andreped** [andreped/GradientAccumulator#31]
The accumulate methods lie in `gradient_accumulate/accumulators.py`. However, it is annoying to import by: `from gradient_accumulate.accumulators import GradientAccumulateOptimizer`. It would be better if we could just: `from gradient_accumulate import GradientAccumulateOptimizer`.

My idea was to add the redirect in the `__init__.py` by adding `from .accumulators import GradientAccumulateOptimizer`. However, that does not seem to be enough, as we still get errors when running the CIs on the cloud. I've also tried to add `from gradient_accumulator.accumulators import GradientAccumulateOptimizer`, but same result...

Annoyingly, it all _seems_ to work fine locally.

This is the verbose of pytest for one of the last builds (see last job [here](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807)):
```
==================================== ERRORS ====================================
[21](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:22)
__________________ ERROR collecting tests/test_train_mnist.py __________________
[22](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:23)
ImportError while importing test module '/home/runner/work/GradientAccumulator/GradientAccumulator/tests/test_train_mnist.py'.
[23](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:24)
Hint: make sure your test modules/packages have valid Python names.
[24](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:25)
Traceback:
[25](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:26)
/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/importlib/__init__.py:127: in import_module
[26](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:27)
    return _bootstrap._gcd_import(name[level:], package, level)
[27](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:28)
tests/test_train_mnist.py:4: in <module>
[28](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:29)
    from gradient_accumulator import GradientAccumulateModel
[29](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:30)
E   ImportError: cannot import name 'GradientAccumulateModel' from 'gradient_accumulator' (/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/gradient_accumulator/__init__.py)
[30](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:31)
------------------------------- Captured stderr --------------------------------
[31](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:32)
2023-01-29 21:16:11.551860: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.7.15/x64/lib
[32](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:33)
2023-01-29 21:16:11.551909: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
[33](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:34)
=============================== warnings summary ===============================
[34](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:35)
../../../../../opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/flatbuffers/compat.py:19
[35](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:36)
  /opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/flatbuffers/compat.py:19: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
[36](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:37)
    import imp
[37](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:38)

[38](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:39)
-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
[39](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:40)
=========================== short test summary info ============================
[40](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:41)
ERROR tests/test_train_mnist.py
[41](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:42)
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
[42](https://github.com/andreped/GradientAccumulator/actions/runs/4038695982/jobs/6942879807#step:8:43)
========================= 1 warning, 1 error in 2.56s ==========================
```

**Comment by andreped** [comment]
@dbouget you have any idea why that doesn't work?

**Comment by andreped** [comment]
For the `focal-loss` package they seem to have gotten in working. See [README](https://github.com/artemmavrin/focal-loss/blob/master/README.rst) and [class](https://github.com/artemmavrin/focal-loss/blob/master/src/focal_loss/__init__.py#L2).

Hmm, is it maybe because they bundle the `tests` inside the `src/focal_loss/` directory?

**Comment by andreped** [comment]
Fixed in af429bd247d681c441f8fbb4f02a2e54fc963411.

I had forgotten `__init__.py` in the `tests/` directory...
